src/Expander.hs

```diff
@@ -411,8 +411,34 @@ initializeKernel = do
             \(ValueString str2) ->
               ValueString (str1 <> str2)
         )
+      , ( "integer->string"
```

Comment:
`int-ops` tests everything but `integer->string`, is that intentional?

Reply:
Nope, that was an oversight. Thanks!